docker-images/kafka/kafka-thirdparty-libs/2.1.x/pom.xml

```diff
@@ -3,7 +3,7 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <groupId>io.strimzi</groupId>
-    <version>0.13.0-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
     <modelVersion>4.0.0</modelVersion>
```

Comment:
Why this jump to 1.0.0?

Reply:
This was originally setup to have the same version as the Strimzi project. But it is not part of the main Maven build it is not updated automatically. It also doesn't make sense for this to have the same version as Strimzi it self when it is not plugged into the main Maven project. So to avoid confusion, I changed the version to _neutral_ `1.0.0-SNAPSHOT`. Do you have any better idea what version should it have?